src/components/SocialFollow.js

```diff
@@ -1,41 +1,48 @@
 import React, { Component } from 'react';
-//import Modal from "react-bootstrap/Modal";
-//import Button from "react-bootstrap/Button";
 import '../css/SocialFollow.css';
-import '../css/Global.css';
 import "bootstrap/dist/css/bootstrap.min.css";
 
-//const fbIcon = "../images/fbIcon.png";
 const fbLink = "https://www.facebook.com/DynamicsPerformanceTeam/";
 
-//const igIcon = "../images/igIcon.png";
 const igLink = "https://www.instagram.com/dynamicsperformanceteam/?hl=en";
 
-//const mailIcon = "../images/mailIcon.png";
 const mailLink = "mailto:dynamicsperformanceteam@gmail.com?subject=Hi, Dynamics Performance Team!";
 
 class SocialFollow extends Component{
     render(){
         return(
             <div className="Social">
               <ul>
+
+              <li>
+                  <a href={fbLink} title="Youtube" target="_blank" rel="noopener noreferrer">
+                    <img className="youtube-iconImg" src={require("../images/youtubeIcon.png")} width="30px" height="30px" alt="error loading img"></img>
+                  </a>
+              </li>
+
+              <li>
+                  <a href={fbLink} title="Yelp" target="_blank" rel="noopener noreferrer">
+                    <img className="yelp-iconImg" src={require("../images/yelpIcon.png")} width="30px" height="30px" alt="error loading img"></img>
+                  </a>
+              </li>
+
               <li>
-                  <a href={fbLink} target="_blank" rel="noopener noreferrer">
-                    <img src={require("../images/fbIcon.png")} width="30px" height="30px" alt="error loading img"></img>
+                  <a href={fbLink} title="Facebook" target="_blank" rel="noopener noreferrer">
+                    <img className="facebook-iconImg" src={require("../images/fbIcon.png")} width="30px" height="30px" alt="error loading img"></img>
                   </a>
               </li>
               <li>
-                  <a href={igLink} target="_blank" rel="noopener noreferrer">
-                    <img src={require("../images/igIcon.png")} width="30px" height="30px" alt="error loading img"></img>
+                  <a href={igLink} title="Instagram" target="_blank" rel="noopener noreferrer">
+                    <img className="instagram-iconImg" src={require("../images/igIcon.png")} width="30px" height="30px" alt="error loading img"></img>
                   </a>
                 </li>
                 <li>
-                  <a href={mailLink} target="_blank" rel="noopener noreferrer">
-                    <img src={require("../images/mailIcon.png")} width="22px" height="22px" alt="error loading img"></img>
+                  <a href={mailLink} title="Email Us" target="_blank" rel="noopener noreferrer">
+                    <img className="mail-iconImg" src={require("../images/mailIcon.png")} width="22px" height="22px" alt="error loading img"></img>
```

Comment:
Is there a reason that the mailIcon.png is set to 22 x 22 pixels and the others are all 30 x 30 pixels?

Reply:
It's minor, but I'm trying to find reasons for why the mail icon's horizontal spacing (padding? margins?) is inconsistent from the rest... (shown below)

<img width="341" alt="Screen Shot 2020-06-19 at 2 52 21 PM" src="https://user-images.githubusercontent.com/37908451/85182085-7c374400-b23c-11ea-9bdb-448a0a6f554e.png">

Reply:
I didn't make any changes to SocialFollows.js aside from adding in styling for the Subscribe button. Mailicon was already set to be 22 x 22 pixels, I'm guessing because making it any bigger explodes the icon on the screen.

Reply:
I also don't recall making changes for the mail icon size, I believe that was added by you @ubhattac, correct me if I'm wrong. My guess as to why is as good as Amrit's.

Reply:
For the comment about the mail icon spacing, the issue is that the Instagram icon wasn't rendered at all for some reason. That's not happening on my end, is it still happening on yours? @ubhattac

Reply:
yeah my bad guys, I just recalled it was made by me (22x22) and it was done for the reason Amrit mentioned. 

@jayrc7 yeah the ig icon isn't rendering... so weird.

Reply:
Alright sounds good, I wasn't able to reproduce the error. I tried zooming in and out on different screen sizes but all the icons are showing. Also, from looking at the css there doesn't seem to be any rules that'd make it disappear. If possible could you give your screen dimensions. Also, try deleting your cache and rerunning npm. @ubhattac